pkg/generators/markers.go

```diff
@@ -61,6 +63,30 @@ func (c *CELTag) Validate() error {
 	return nil
 }
 
+func isKnownTagCommentKey(key string) bool {
```

Comment:
Could you comment this - Would be nice to explain the expected "grammar" for a key here.

Reply:
Sure, added.

Reply:
not pushed?

Reply:
oops, pushed now